src/icon/icon-template.txt

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+// BASEUI-GENERATED-REACT-ICON
```

Comment:
should we also explicitly state that one should not modify it manually?

Reply:
good call, will update.